src/components/AccessibilityGuide/ProgressiveEnhancement.mdx

```diff
@@ -0,0 +1,229 @@
+import { Meta, Source } from "@storybook/blocks";
+
+import { Banner } from "../Banner/Banner";
+import Image from "../Image/Image";
+import Link from "../Link/Link";
+
+import progressiveEnhancementExample from "/static/progressive-enhancement.png";
+import { exampleWrapperStyles } from "../../utils/utils.ts";
+
+<Meta title="Accessibility Guide/Progressive Enhancement" />
+
+# Progressive Enhancement
+
+## Table of Contents
+
+- {<Link href="#general-information" target="_self">General Information</Link>}
+- {<Link href="#nypl-and-ds-problem" target="_self">NYPL and DS Problem</Link>}
+- {<Link href="#html-button-issue" target="_self">HTML Button Issue</Link>}
+- {<Link href="#component-audit" target="_self">Component Audit</Link>}
+- {<Link href="#general-recommendations" target="_self">General Recommendations</Link>}
+- {<Link href="#resources" target="_self">Resources</Link>}
+
+## General Information
+
+Progressive enhancement is a frontend development strategy and mindset that
+emphasizes building a "basic", functional version of a website or component first,
+and then adding more advanced features and enhancements. This approach ensures
+that the core functionality is accessible to all users, regardless of their
+device, browser capabilities, or connection quality, rather than focusing on the
+few who intentionally disable JavaScript.
+
+A key benefit of progressive enhancement is that it prioritizes a fast and
+reliable first render of the page. By ensuring the base HTML delivers meaningful
+content and functionality upfront, we can improve usability and equity for users
+in bandwidth-constrained environments. Once the foundation is in place, we can
+then provide additional enhancements for users with modern browsers and devices.
+
+On the development side, this inclusive design philosophy focuses on knowing and
+implementing web fundamentals, such as HTML and CSS, before heavily relying on
+JavaScript. Client-side JavaScript for core functionality places the burden on
+the user's device and browser to download, parse, and execute JavaScript code.
+
+## NYPL and DS Problem
+
+When JavaScript is turned off in the browser, certain NYPL user interfaces and
+features do not work. There are various reasons why this is, including how the
+application is architected, how data is fetched, and how data is rendered. Many
```

Comment:
and also whether no-javascript functionality is supported by vendors that we are using.